### ISSUE TITLE
Onnx export how do i interpret output

### ISSUE THREAD

**Issue by velnz08** [voldemortX/pytorch-auto-drive#181]
How to use the output of the resnet18s_lstr_tusimple_20210701.onnx?
The input is straightforward i think, but the output is composed of "curves" and "logits" and the got me trying a lot of different visualization methods with no success so far, help please!

**Comment by voldemortX** [comment]
> How to use the output of the resnet18s_lstr_tusimple_20210701.onnx?
> The input is straightforward i think, but the output is composed of "curves" and "logits" and the got me trying a lot of different visualization methods with no success so far, help please!
> 

You can refer to the visualization codes within this repo.
